library/src/scripts/headers/mebox/pieces/UserDropDownContents.tsx

```diff
@@ -41,14 +41,23 @@ function UserDropDownContents(props: IProps) {
             <DropDownUserCard className="userDropDown-userCard" />
             <DropDownItemSeparator />
             <DropDownItemLink to="/profile/edit" name={t("Edit Profile")} />
+            <DropDownSection title={t("Articles")}>
+                <Permission permission="articles.add">
```

Comment:
The permission should be on the outside of the section, or else it will still render HTML.